### PR TITLE
feat(monorepo): Add contributing guide

### PR DESCRIPTION
**Pull request by michelguerin**
Details the installation steps to contribute as a developer.

* Link from readme to contributing guide + refacto
* Add .env.example file for needed environment variables
* Add `.env` to `.gitignore`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Think Before Adding a Dependency
+
+Most deps should be added to `devDependencies` even if they are needed at runtime. Some exceptions are:
+
+- Type packages. Example: `@types/*`.
+- Deps that cannot be properly bundled due to binary files. Example: `esbuild`.
+- Deps that ship their own types that are used in Vite's own public types. Example: `rollup`.
+
+Avoid deps with large transitive dependencies that result in bloated size compared to the functionality it provides. For example, `http-proxy` itself plus `@types/http-proxy` is a little over 1MB in size, but `http-proxy-middleware` pulls in a ton of dependencies that make it 7MB(!) when a minimal custom middleware on top of `http-proxy` only requires a couple of lines of code.
```
Comment (clepski) [inline CONTRIBUTING.md:79]: Copy paste from Vite? There are severals mentions to Vite

Comment (michelguerin) [reply CONTRIBUTING.md]: Indeed, I use the vitejs contributing guide as template for this one ;)

**Review by clepski — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -7,13 +7,13 @@
 
 Open Substation Communication Designer is an editor for SCL files as described in `IEC 61850-6`.
 
-> Try it out at [openscd.github.io](https://openscd.github.io)!
+> Try it out at [↗ openscd.github.io](https://openscd.github.io)!
```
Comment (trusz) [inline README.md:10]: thank you!

**Comment by trusz** [comment]
We already have a `CONTRIBUTING.md` in the core package: https://github.com/openscd/open-scd/blob/main/packages/core/CONTRIBUTING.md

Have you took that into consideration?

**Comment by michelguerin** [comment]
> We already have a `CONTRIBUTING.md` in the core package: https://github.com/openscd/open-scd/blob/main/packages/core/CONTRIBUTING.md
> 
> Have you took that into consideration?

Oups, didn't see this. I'll merge these to file in favor of the root one. I'll make a new proposition in the next commit.

**Pull request merged**